app/models/theme_field.rb

```diff
@@ -138,6 +138,8 @@ def process_extra_js(content)
         js_compiler.append_module(content, filename)
       when "hbs"
         js_compiler.append_ember_template(filename.sub("discourse/templates/", ""), content)
+      when "hbr", "raw.hbs"
+        js_compiler.append_raw_template(filename.sub("discourse/templates/", ""), content)
       when "raw.hbs"
```

Comment:
You should remove lines 143 and 144 as they will never be executed 😉

Reply:
lol thank you again